docs/docs/overview/quickstart/installation.md

```diff
@@ -108,14 +110,15 @@ $ ./bp
 ```
 
 ### Alternative - Commands
+<!-- for me this would go up to the previous paragraph -->
```

Comment:
question 5. 

I'd like tfo merge the two lines into "Starting Botpress". For me when looking at it again, the terminal and the CLI are the same thing. We just need to add the --help command above. https://github.com/botpress/v12/pull/1743/files#diff-ddef161647c824f425001687d86a721502ab5490ab90c9f26248e9e461df2f02R38

Reply:
```suggestion
```